tokio-process/src/windows.rs

```diff
@@ -18,12 +18,22 @@
 extern crate mio_named_pipes;
 extern crate winapi;
 
+use crate::kill::Kill;
+
 use std::fmt;
+use std::future::Future;
 use std::io;
 use std::os::windows::prelude::*;
 use std::os::windows::process::ExitStatusExt;
+use std::pin::Pin;
 use std::process::{self, ExitStatus};
 use std::ptr;
+use std::task::Context;
+use std::task::Poll;
+
+use futures::channel::oneshot;
```

Comment:
We've been using `tokio_sync`'s channel implementation within other tokio crates, so might be good to keep it consistent here too